src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -2074,7 +2074,6 @@ public void onFailure(Throwable e)
                     }
                     logger.error(message, e);
                     progressSupport.progress("bootstrap", new ProgressEvent(ProgressEventType.ERROR, 1, 1, message));
-                    progressSupport.progress("bootstrap", new ProgressEvent(ProgressEventType.COMPLETE, 1, 1, "Resume bootstrap complete"));
```

Comment:
The thing that worries me about this is that it will hang older versions of `nodetool` when an error occurs. They will be waiting for a `COMPLETE` that will never arrive after printing the error message. The fact that we're changing the behavior of `nodetool` itself on error in this Jira probably means that we only need to patch trunk, but it's probably not worth it to remove the `COMPLETE` here.